src/PSRule.Rules.Azure/rules/Azure.AppService.Rule.ps1

```diff
@@ -155,19 +155,68 @@ Rule 'Azure.AppService.WebProbePath' -Ref 'AZR-000080' -With 'Azure.AppService.I
 # Synopsis: Web apps should disable insecure FTP and configure SFTP when required.
 Rule 'Azure.AppService.WebSecureFtp' -Ref 'AZR-000081' -With 'Azure.AppService.IsWebApp' -Tag @{ release = 'GA'; ruleSet = '2022_06'; 'Azure.WAF/pillar' = 'Security'; } -Labels @{ 'Azure.MCSB.v1/control' = 'DP-3' } {
     $siteConfigs = @(GetWebSiteConfig | Where-Object {
-        $Assert.HasField($_, 'Properties.ftpsState').Result
-    });
+            $Assert.HasField($_, 'Properties.ftpsState').Result
+        });
     if ($siteConfigs.Length -eq 0) {
         return $Assert.In($TargetObject, 'Properties.siteConfig.ftpsState', @(
-            'FtpsOnly'
-            'Disabled'
-        ));
+                'FtpsOnly'
+                'Disabled'
+            ));
     }
     foreach ($siteConfig in $siteConfigs) {
         $Assert.In($siteConfig, 'Properties.ftpsState', @(
-            'FtpsOnly'
-            'Disabled'
-        ));
+                'FtpsOnly'
+                'Disabled'
+            ));
+    }
+}
+
+# Synopsis: Use supported Node.js versions for the runtime of applications.
+Rule 'Azure.AppService.MigrateNodeJs' -Ref 'AZR-000425' -Type 'Microsoft.Web/sites', 'Microsoft.Web/sites/config', 'Microsoft.Web/sites/slots', 'Microsoft.Web/sites/slots/config' -Tag @{ release = 'GA'; ruleSet = '2024_06'; 'Azure.WAF/pillar' = 'Reliability'; } {
+    
+    [SemVer[]]$versions = @(
```

Comment:
`SemVer` is a PowerShell 7.x specific type. It doesn't work on Windows PowerShell.

Until we drop support for Windows PowerShell we need to keep compatibility.

But `$Assert.Version` supports parsing semver from a string, so I don't think there is a reason to use to type specifically.

Reply:
Good news! Thanks.